### ISSUE TITLE
Bindable Values not getting notified after Refresh

### ISSUE THREAD

**Issue by ermanishks** [evant/binding-collection-adapter#220]
Is this a general databinding issue or question? You are more likely to get a
response from somewhere like
[StackOverflow](https://stackoverflow.com/questions/tagged/android-databinding).

## Issue
Hello All, 
I am facing a strange issue  with BindingRecyclerViewAdapter. 

When list gets refresh ( pull to refresh or from server ) , if bindViewHolder doesn't called ( since there are exactly same data , i am using DiffcallBack),  Bindable property stopped working . 
To put it simply , on refresh of list , notifyPropertyChanged(Br.property) is not getting notified. 
I can supply with the detail code snippet. 

@evant have you faced something like this earlier ?
